flang/test/Semantics/OpenMP/do08.f90

```diff
@@ -61,7 +61,6 @@ program omp
   !$omp end do
 
 
-  !ERROR: Canonical loop nest must be perfectly nested.
```

Comment:
These ERRORs were added in #60283. Those are still non-perfectly nested loops, but the other errors get precedence.